lightning/src/routing/router.rs

```diff
@@ -4106,15 +4107,39 @@ mod tests {
 		{
 			// Attempt to route more than available results in a failure.
 			if let Err(LightningError{err, action: ErrorAction::IgnoreError}) = get_route(
-					&our_id, &payment_params, &network_graph.read_only(), None, 300_000, 42, Arc::clone(&logger), &scorer, &random_seed_bytes) {
-				assert_eq!(err, "Failed to find a sufficient route to the given destination");
+				&our_id, &payment_params, &network_graph.read_only(), None, 300_000, 42,
+				Arc::clone(&logger), &scorer, &random_seed_bytes) {
+					assert_eq!(err, "Failed to find a sufficient route to the given destination");
+			} else { panic!(); }
+		}
+
+		{
+			// Attempt to route while setting max_mpp_path_count to 0 results in a failure..
```

Comment:
Nit: double period here and below.

Reply:
Fixed with https://github.com/lightningdevkit/rust-lightning/pull/1526/commits/4617555d888f62e664b85fe2a9e34ae6e4d81fd2.